config/locales/models/en.yml

```diff
@@ -155,6 +155,13 @@ en:
               not_work: Only a link to a work can be listed as an inspiration.
               protected: You can't use the related works function to cite works by the protected user %{login}.
           format: "%{message}"
+        skin:
+          archive_in_title: You can't use the word 'Archive' in your skin title, sorry! (We have to reserve it for official skins.)
+          attributes:
+            title:
+              taken: must be unique
+          invalid_media: We don't currently support the media type %{media}, sorry! If we should, please let Support know.
+          no_public_preview: You need to upload a screencap if you want to share your skin.
```

Comment:
Noting that we would update the wording here, but at some point this feature is going away anyways so it's not really worth updating only to delete later